mysql-compatibility.md

```diff
@@ -9,7 +9,7 @@ aliases: ['/docs/dev/mysql-compatibility/','/docs/dev/reference/mysql-compatibil
 
 TiDB is fully compatible with the MySQL 5.7 protocol and the common features and syntax of MySQL 5.7. The ecosystem tools for MySQL 5.7 (PHPMyAdmin, Navicat, MySQL Workbench, mysqldump, and Mydumper/myloader) and the MySQL client can be used for TiDB.
 
-However, some features of MySQL are not supported. This could be because there is now a better way to solve the problem (such as XML functions superceded by JSON), security issues (such as `SELECT INTO OUTFILE`), or a lack of current demand versus effort required (such as stored procedures and functions). Some features might also be difficult to implement as a distributed system.
```

Comment:
`select into outfile` is supported in TiDB 4.0, we'd better not backport this PR to release 2.1, 3.0, and 3.1.

Reply:
Understood. The 2.1-3.0 docs will still say this feature is not supported (it's further down in the document). What is removed is an explanation of why we will not add certain features such as this one (which becomes confusing, since it was added in a later version).